src/location.rs

```diff
@@ -69,6 +70,16 @@ pub fn read_locations(root_path: &str) -> io::Result<Vec<LocationEntry>> {
     Ok(packets)
 }
 
+pub fn get_local_location_id(root_path: &str) -> Result<String, Error> {
```

Comment:
```suggestion
pub fn get_local_location_id(root_path: &str) -> io::Result<String> {
```

(slightly less verbose way of writing this that we have used elsewhere)

Reply:
thanks!